schema/deploy/shipping/views.sql

```diff
@@ -1013,6 +1014,74 @@ grant select
    to "incidence-modeler";
 
 
+create or replace view shipping.phskc_encounter_details_v1 as
+
+    with
+        phskc_encounters as (
+            select encounter_id,
+                age,
+                sex
+            from warehouse.encounter
+              left join warehouse.individual using (individual_id)
+            where site_id = 569
+            group by encounter_id, sex
+        ),
+
+        encounter_metadata as (
+            select encounter_id,
+                max (code_response[1]) filter ( where link_id = 'race' ) as race,
+                max (string_response[1]) filter ( where link_id = 'survey_testing_because_exposed' ) as survey_testing_because_exposed,
+                max (string_response[1]) filter ( where link_id = 'if_symptoms_how_long' ) as if_symptoms_how_long,
+                max (string_response[1]) filter ( where link_id = 'vaccine_status' ) as vaccine_status,
+                bool_or (boolean_response) filter ( where link_id = 'ethnicity' ) as hispanic_or_latino,
+                bool_or (boolean_response) filter ( where link_id = 'inferred_symptomatic' ) as inferred_symptomatic,
+                bool_or (boolean_response) filter ( where link_id = 'survey_have_symptoms_now' ) as survey_have_symptoms_now
+            from shipping.fhir_questionnaire_responses_v1
+            group by encounter_id
+        ),
+
+        location_info as (
+            select encounter_id,
+                hierarchy as loc_data
+            from warehouse.encounter
+              left join warehouse.encounter_location using (encounter_id)
+              left join warehouse.location using (location_id)
+        ),
+
+        pa_result as (
+            select encounter_id,
+                present as present
+            from warehouse.encounter
+              left join warehouse.sample using (encounter_id)
+              left join warehouse.presence_absence as pa using (sample_id)
+            where target_id = 952
+            group by encounter_id, present
+        )
+
+    select
+        encounter_id,
+        extract(years from age) as age,
```

Comment:
Could use the `age_in_years(interval)` function here for consistency with other shipping views.